src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt

```diff
@@ -0,0 +1,45 @@
+package com.hibob.academy.employeeFeedback.service
+
+import com.hibob.academy.employeeFeedback.dao.*
+import jakarta.ws.rs.BadRequestException
+import org.springframework.stereotype.Service
+
+@Service
+class FeedbackService(private val feedbackDao: FeedbackDao, private val employeeDao: EmployeeDao) {
+    fun submitFeedback(employeeId: Int, anonymity: AnonymityType, feedback: String): Boolean {
+        val employeeInfo = employeeDao.getEmployeeById(employeeId)
```

Comment:
This is something that we discussed offline, think of whether you really need to fetch this data here or you can receive it from the cookie and then forward it as a parameter to the service.

The rule of thumb should be - minimizing unnecessary queries

Reply:
I was sure I deleted this. I'll change it